examples/jobs/poll_job.py

```diff
@@ -0,0 +1,61 @@
+"""Example script for polling a job from the Jobs API."""
```

Comment:
The examples should be very practical examples. I would write an example script where you create a job. Then poll the job until it is completed then run another action. I would also structure the polling function the tamr-toolbox `wait` function here: https://github.com/Datatamer/tamr-toolbox/blob/develop/tamr_toolbox/utils/operation.py#L135